### ISSUE TITLE
Jasmine D. Orientation

### ISSUE THREAD

**Issue by dkbaldwin** [inDevelopme/indevelopme-coaching-publications#67]
Will complete the following topics under the JustAskDavidB [blog](https://medium.com/indevelopme-tech-coaching-program):

## Articles to Read
- [x] JustAskDavidB Coaching Program
- [x] Orientation Packet
- [x] The Dunn-Kruger Effect
- [x] Introduction to Queues and Stacks Structure

## Programs to install
Install following programs on computer:

- [x] [Pycharm Community Version ](https://www.jetbrains.com/pycharm/download)
- [x] [Docker Desktop](https://www.docker.com/products/docker-desktop/)
- [x] [Github Desktop](https://desktop.github.com/)

## Accounts to create
Create accounts in the following 3rd Party applications:

- [x] Github account
- [x] Linkedin
- [x] Discord (we will add you to the discord group later)
- [x] Medium (free account only)

Estimated Time: 5-8 hours

**Comment by jasalise** [comment]
Will add lesson learned below:

**Comment by jasalise** [comment]
Docker Desktop required apple processor version, as did Pycharm. And I installed the professional version of Pycharm, BUT anyone can install the community version to save money. I was able to install Github Desktop but cannot yet run git command from the terminal because I have to install Xcode which claims it will take 64 hours! Recommend a template be created for first time players that has "pycharm community" directly linked because the link above takes you to the professional version.
